src/pages/workspace/reportFields/ReportFieldsListValuesPage.tsx

```diff
@@ -179,8 +179,6 @@ function ReportFieldsListValuesPage({
         }
 
         Navigation.navigate(ROUTES.WORKSPACE_REPORT_FIELDS_VALUE_SETTINGS.getRoute(policyID, valueItem.index, reportFieldID));
-
-        setSelectedValues({});
```

Comment:
This page still have the deletion count bug.


https://github.com/user-attachments/assets/772963bc-c73e-4ae3-b31f-cd4eadd3222a